.github/workflows/ci-pip-tools.yml

```diff
@@ -39,6 +39,7 @@ jobs:
           git init
           pip install --upgrade pip
           pip install -r dev-requirements.txt
+          pip install -r doc-requirements.txt
           pip install -e .
```

Comment:
```suggestion
          pip install -e .[dev,doc]
```

Reply:
Ahhh ok.

I guess it's hard to know how restrictive you want to be with pinned dependencies. `poetry` only pins the versions of direct dependencies for published packages, I think. You probably wouldn't want to pin versions of all your indirect dependencies, otherwise you might not be able to install your package in a virtualenv alongside something else with slightly different version requirements.